drivers/sensors/adns5050.c

```diff
@@ -211,3 +219,8 @@ bool adns5050_check_signature(void) {
 
     return (pid == 0x12 && rid == 0x01 && pid2 == 0x26);
 }
+
+void adns5050_power_down(void) {
+    powered_down = true;
+    adns5050_write_reg(REG_MOUSE_CONTROL, 0b10);
```

Comment:
Suspend_power_down gets called repeatedly. Just to minimize calls...
```suggestion
    if (!powered_down) {
        powered_down = true;
        adns5050_write_reg(REG_MOUSE_CONTROL, 0b10);
    }
```